BUILDING_ON_WINDOWS_WITH_VCPKG.md

```diff
@@ -29,7 +29,7 @@ This will require more than 30GB of free space on your hard drive.
      See [VCPKG_ROOT documentation](https://learn.microsoft.com/en-gb/vcpkg/users/config-environment#vcpkg_root)
    - Append the vcpkg path to your path  
      e.g. `setx PATH "%PATH%;<path to vcpkg>"`
-   - For more configurations, see https://learn.microsoft.com/en-gb/vcpkg/users/config-environment
+   - For more configurations, see <https://learn.microsoft.com/en-gb/vcpkg/users/config-environment>
```

Comment:
what does this syntax do?

Reply:
Although GitHub Markdown recognizes the link either way, [CommonMark](https://spec.commonmark.org/0.31.2/#autolinks) requires the link to be wrapped in `<>`. This basically fixes [MD034](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md034.md).

Reply:
kk that's fine then :+1: